src/PrestaShopBundle/Resources/config/services/adapter/product/_image.yml

```diff
@@ -69,7 +69,6 @@ services:
 
   PrestaShop\PrestaShop\Adapter\Product\Image\Repository\ProductImageMultiShopRepository:
     autowire: true
-    public: false
```

Comment:
why?

Reply:
Because it's used in a behat context (better than relying on legacy static methods), it used to rely on `ProductImageRepository` I replaced it with `ProductImageMultiShopRepository` because I anticipate the PR that aims at merging both together anyway

Reply:
It's used here https://github.com/PrestaShop/PrestaShop/blob/59d9d1af6cf754b4fe75445a77a312b0a3dd0016/tests/Integration/Behaviour/Features/Context/Domain/Product/ProductImageFeatureContext.php#L70

Reply:
Yes you're right it's not the appropriate PR to change things, it was just a question to understand. I think we should consider the following library to inject services in behat context: https://packagist.org/packages/friends-of-behat/symfony-extension